queries/appSettings.ts

```diff
@@ -83,21 +44,22 @@ export async function requireAppNotExpired(isSetupRoute = false) {
     return;
   }
 
-  if (!configured) {
+  const isConfigured = await getAppSetting('configured');
+
+  if (!isConfigured) {
     redirect('/setup');
   }
 
   return;
 }
 
 export async function isAppExpired() {
-  const configured = await getAppSetting('configured');
+  const isConfigured = await getAppSetting('configured');
   const initializedAt = await getAppSetting('initializedAt');
 
-  if (!configured || !initializedAt) {
-    throw new Error('Could not get app settings');
-  }
-  return calculateIsExpired(configured, initializedAt);
+  return (
+    !isConfigured && initializedAt.getTime() < Date.now() - UNCONFIGURED_TIMEOUT
```

Comment:
This is throwing an error on resetting the db and on changing setup steps that `initializedAt.getTime()` is not a function. I think that there's a problem in the way getAppSetting is preprocessing the value as a date.

Reply:
Looked into this more and it is being correctly preprocessed. The only way I can get this to work is to explicitly create a new `Date` in this function before using `getTime`. Implemented here [9cca019](https://github.com/complexdatacollective/Fresco/pull/162/commits/9cca019e3b866a31ee57b830f2abdd5e67bef01e)